TODO

```diff
@@ -0,0 +1,10 @@
+- docker-compose -f
```

Comment:
Same here do we want to store such files in the project repo 🤔? If so I'd say that maybe the README.md file is a place for that, wdyt?

Reply:
Yeah, I agree. But in order to put that inside README it'd need to be in English and I didn't want to spend time on that since hopefully all of those problems will have been resolved by the end of the next phase.

Reply:
Sure that makes sense, thanks for the explanation.